public/img/Tip1.svg

```diff
@@ -1,86 +0,0 @@
-				c-0.789-2.944-3.089-5.244-6.033-6.033c-2.944-0.789-6.086,0.053-8.242,2.208l-19.567,19.567l-2.5-2.5
-				c-3.336-3.308-8.719-3.296-12.041,0.026c-3.322,3.322-3.333,8.705-0.025,12.041L241.434,185.279z"/>
-			<path d="M145.067,213.333c-14.132,0.015-25.585,11.468-25.6,25.6V307.2c0.015,14.132,11.468,25.585,25.6,25.6h221.867
-				c14.132-0.015,25.585-11.468,25.6-25.6v-68.267c-0.015-14.132-11.468-25.585-25.6-25.6H332.8c-4.713,0-8.533,3.82-8.533,8.533
-				s3.821,8.533,8.533,8.533h34.133c4.711,0.005,8.529,3.822,8.533,8.533V307.2c-0.005,4.711-3.822,8.529-8.533,8.533H145.067
-				c-4.711-0.005-8.529-3.822-8.533-8.533v-68.267c0.005-4.711,3.822-8.529,8.533-8.533H179.2c4.713,0,8.533-3.821,8.533-8.533
-				s-3.82-8.533-8.533-8.533H145.067z"/>
-			<path d="M204.363,267.232c0.716-2.147,0.549-4.491-0.463-6.516c-1.012-2.025-2.786-3.565-4.934-4.281
-				c-2.148-0.716-4.492-0.549-6.516,0.464l-4.717,2.358V256c0-4.713-3.82-8.533-8.533-8.533s-8.533,3.82-8.533,8.533v3.258
-				l-4.717-2.358c-2.729-1.375-5.984-1.187-8.536,0.494c-2.552,1.681-4.01,4.598-3.824,7.647s1.989,5.767,4.726,7.125l1.8,0.9
-				l-1.8,0.9c-4.2,2.117-5.896,7.232-3.792,11.438s7.213,5.919,11.426,3.829l4.717-2.358v3.258c0,4.713,3.82,8.533,8.533,8.533
-				s8.533-3.821,8.533-8.533v-3.258l4.717,2.358c4.213,2.09,9.323,0.378,11.426-3.829s0.407-9.321-3.792-11.438l-1.8-0.9l1.8-0.9
-				C202.108,271.155,203.648,269.38,204.363,267.232z"/>
-			<path d="M281.163,267.232c0.716-2.147,0.549-4.491-0.463-6.516c-1.011-2.026-2.786-3.567-4.933-4.283
-				c-2.148-0.716-4.492-0.548-6.517,0.466l-4.717,2.358V256c0-4.713-3.82-8.533-8.533-8.533s-8.533,3.82-8.533,8.533v3.258
-				l-4.717-2.358c-2.729-1.375-5.984-1.187-8.536,0.494c-2.552,1.681-4.01,4.598-3.824,7.647c0.186,3.05,1.989,5.767,4.726,7.125
-				l1.8,0.9l-1.8,0.9c-4.2,2.117-5.896,7.232-3.792,11.438s7.213,5.919,11.426,3.829l4.717-2.358v3.258
-				c0,4.713,3.82,8.533,8.533,8.533s8.533-3.821,8.533-8.533v-3.258l4.717,2.358c4.213,2.09,9.323,0.378,11.426-3.829
-				s0.407-9.321-3.792-11.438l-1.8-0.9l1.8-0.9C278.908,271.155,280.448,269.38,281.163,267.232z"/>
-			<path d="M311.014,257.394c-2.552,1.681-4.01,4.598-3.824,7.647c0.186,3.05,1.989,5.767,4.726,7.125l1.8,0.9l-1.8,0.9
-				c-4.2,2.117-5.896,7.232-3.792,11.438s7.213,5.919,11.426,3.829l4.717-2.358v3.258c0,4.713,3.821,8.533,8.533,8.533
-				c4.713,0,8.533-3.821,8.533-8.533v-3.258l4.717,2.358c4.213,2.09,9.323,0.378,11.426-3.829c2.103-4.206,0.407-9.321-3.792-11.438
-				l-1.8-0.9l1.8-0.9c4.2-2.117,5.896-7.232,3.792-11.438c-2.103-4.206-7.213-5.919-11.426-3.829l-4.717,2.358V256
-				c0.001-2.263-0.898-4.434-2.499-6.035c-1.6-1.6-3.771-2.499-6.035-2.499c-2.263-0.001-4.434,0.898-6.035,2.499
-				c-1.6,1.6-2.499,3.771-2.499,6.035v3.258l-4.717-2.358C316.821,255.525,313.566,255.713,311.014,257.394z"/>
-		</g>
-	</g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-</svg>

```

Comment:
Out of curiosity, why the switch from SVG to PNG? Was it purely a file size thing?

Reply:
Not sure on this. The previous SVGs didn't match up with the icons used in the spec, but I'm not sure what brought about the PNG decision.